src/types.ts

```diff
@@ -253,6 +256,7 @@ export interface IEmailProperties {
   createdModSeq: number;
   updatedModSeq: number;
   receivedAt: IUtcDate;
+  headers?: EmailHeader[];
```

Comment:
Maybe

```suggestion
  headers: EmailHeader[] | null;
```

To be consistent?

Reply:
Yes, you are right.